echopype/calibrate/calibrate_ek.py

```diff
@@ -748,10 +750,17 @@ def _cal_complex(self, cal_type, waveform_mode) -> xr.Dataset:
             prx.name = "received_power"
             prx = prx.to_dataset()
 
-        # derived params
-        sound_speed = self.env_params["sound_speed"].squeeze()
-        absorption = self.env_params["sound_absorption"].sel(channel=chan_sel).squeeze()
-        range_meter = self.range_meter.sel(channel=chan_sel).squeeze()
+        # Derived params
+        # TODO: right now squeeze out the single ping_time dimension
+        #  Need to properly align based on actual time
+        #  This ping_time is actually renamed from time1 in Environment group,
+        #  so that in the computation of range_meter it can be aligned with echo data easily.
+        #  This however makes this part of code less traceable,
+        #  since the ping_time here does not come from the "pings"
+        #  Will resolve later.
+        sound_speed = self.env_params["sound_speed"].squeeze(drop=True)
+        absorption = self.env_params["sound_absorption"].sel(channel=chan_sel).squeeze(drop=True)
+        range_meter = self.range_meter.sel(channel=chan_sel)
```

Comment:
I'm curious why you removed the squeeze() here altogether. I assume it's all good, but I'll take a closer look.

Reply:
It is because `range_meter` would usually have `ping_time` with length >1 so `squeeze` here has no effect,  and even if the `ping_time` length is 1, `ping_time` actually should not be removed.

Reply:
Thanks!